src/Core/Engine/Components/Light.hpp

```
@@ -18,91 +18,58 @@ enum class LightType
 
-struct alignas(16) LightBaseGPU
+class DirectionalLight
 {
+public:
     glm::vec3 color = glm::vec3(1.0f, 1.0f, 1.0f);
+    glm::vec3 direction = glm::vec3(0.0f, 0.0f, 0.0f);
+    glm::vec3 ambient = glm::vec3(1.0f, 1.0f, 1.0f);
+    glm::vec3 diffuse = glm::vec3(1.0f, 1.0f, 1.0f) ;
+    glm::vec3 specular = glm::vec3(.05f, .05f, .05f);
+
     float intensity = 1.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)